.github/BOTMETA.yml

```
@@ -223,7 +223,7 @@ files:
   $lookups/dnstxt.py:
     maintainers: jpmens
   $lookups/dsv.py:
-    maintainers: amigus endlesstrax delineaKrehl tylerezimmerman
+    maintainers: delineaKrehl tylerezimmerman
```

You have to add `amigus` to the ignore list since he's author of that plugin. Same below.
```suggestion
    maintainers: delineaKrehl tylerezimmerman
    ignore: amigus
```